codegen/src/main.rs

```diff
@@ -105,3 +114,10 @@ fn codegen(
         std::fs::copy(&path, &to).unwrap();
     }
 }
+
+fn write_fds(fds: &FileDescriptorSet, path: &Path) {
+    let mut writer = BufWriter::new(std::fs::File::create(path).unwrap());
```

Comment:
Nit: I'd suggest importing `File` instead.

Reply:
Applied the suggestion.

Reply:
Applied the suggestion.